server/app/lib/utils/fusor/openstack_dns.rb

```diff
@@ -0,0 +1,60 @@
+require 'fog'
+
+module Utils
+  module Fusor
+    class OpenstackDNS
```

Comment:
instead of passing in openstack_deployment to all these methods, mind if we just add it in the initialize and use the @openstack_deployment in the methods?   Thsn, all the functions to "get_x" would just be more normal Ruby getters, too. 

```ruby
def initialize(openstack_deployment)
  @openstack_deployment = openstack_deployment
end

def compare
  overcloud_dns == satellite_dns && undercloud_dns == satellite_dns
end

...

def undercloud
{ :openstack_auth_url  => "http://#{@openstack_deployment.undercloud_ip_address}:5000/v2.0/tokens",
  :openstack_username  => 'admin', :openstack_tenant => 'admin',
  :openstack_api_key   => @openstack_deployment.undercloud_admin_password }
end
```